BSP/Inc/BSP_Contactors.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __BSP_CONTACTORS_H
+#define __BSP_CONTACTORS_H
+
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+void BSP_Contactors_Init(void);
+void read_Contactor_States(int16_t *contactorStates);
+void BSP_Contactor_Set(int16_t motorContactorState, int16_t arrayContactorState);
```

Comment:
2 things.

1. Be consistent with the plurality of the naming (just for ease of use in the future). So call this BSP_Contactors_Set 
2. So that we don't have to pass both states every time, I would have it take an enum as a parameter (similar to how it's done in BSP_CAN with the 2 different CAN lines). That way you can have it take the enum and a state and it will set the appropriate contactor state and not touch the other one.

Reply:
Changed function name to BSP_Contactors_Set().
Its inputs are now enums for both contactor type and state.